ci-operator/step-registry/openshift/e2e/test/qe/OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
```

Comment:
@liangxia could you please add jianzhangbjz into approvers?
cc @jianzhangbjz

Reply:
Done.
Actually this is more like a one time shot. We will need changes there unless we want to add tests for another repo.